Quentinchampenois/decidim-module-extra_user_fields
# [Feature] - Add Firstname field to extra_user_fields

## 🎩 Description

Addition of firstname field to extra_user_fields to give the possibility to the platform to ask for firstname of the user when registration.

## How to test

- Initialize your instance
- Log as an admin
- Go to settings
- Enable extra user fields
- Activate firstname
- Disconnect
- Try to register
- Fill in infos and fill in firstname
- Register
- Log out and log as an admin
- Export users in backoffice
- Check if in the export there is the firstname of your new user
